src/control/actor/host/notify.go

```diff
@@ -57,27 +53,41 @@ func (c *HostNotifyCmd) listenCloseF(net network.Network, addr ma.Multiaddr) {
 
 func (c *HostNotifyCmd) connectedF(net network.Network, conn network.Conn) {
 	logrus.Info("connection detected: ", conn.RemotePeer().String())
-
-	// request metadata as soon as we connect to a peer
-	peerData, err := PollPeerMetadata(conn.RemotePeer(), c.Base, c.PeerMetadataState, c.Store, c.PeerStore)
+	h, _ := c.Host()
+	// Request the Host Metadata
+	hInfo := ReqHostInfo(context.Background(), h, conn)
+	// Request the BeaconMetadata
+	bMetadata, err := ReqBeaconMetadata(context.Background(), h, conn.RemotePeer())
+	if err != nil {
+		log.Warn(err)
+	}
+	// request BeaconStatus metadata as we connect to a peer
+	bStatus, err := ReqBeaconStatus(context.Background(), h, conn.RemotePeer())
```

Comment:
same, as in 60. the error is not handled so `UpdateBeaconStatus` will be called with wrong information